README.md

```diff
@@ -104,6 +104,7 @@ convert(delta, formats, { blockTag: 'FIGURE', inlineTag: 'INS' });
 
 ## Changelog
 
+- `6.0.0` Remove private formats and internal-specific outputs
```

Comment:
Could you document specifically that `toInternalHtml` has been removed from this version? That way anyone upgrading to v6 can know what changes they may need to make (it looks like that's just anthem, which you've already addressed, but ya know... just to be thorough).

Reply:
good call. I will update this 👍